staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/conversion/converter.go

```diff
@@ -35,14 +35,15 @@ type CRConverterFactory struct {
 	// webhookConverterFactory is the factory for webhook converters.
 	// This field should not be used if CustomResourceWebhookConversion feature is disabled.
 	webhookConverterFactory *webhookConverterFactory
-	converterMetricFactory  *converterMetricFactory
 }
 
+// converterMetricFactorySingleton protects us from reregistration of metrics on repeated
+// apiextensions-apiserver runs.
+var converterMetricFactorySingleton = newConverterMertricFactory()
```

Comment:
@liggitt this fixes metrics reregistration errors because multiple test apiservers had different instances and didn't protect us from calling `prometheus/registry.Register` twice. /cc @logicalhan

Reply:
can you open a follow-up issue for @logicalhan to think through ways to improve this for multi-instance tests? (also to fix the newConverterMertricFactory typo)